### ISSUE TITLE
FA API

### ISSUE THREAD

**Issue by vishalparkar** [FrontAccountingERP/FA#47]
Is it technically and possible and feasible to integrate FA with a web-based user management system written in other technologies (e.g. Django) where the users will enter data in FA through that system ?

**Comment by cambell-prince** [comment]
Yes, there is a REST API available as a module. See [FrontAccountingSimpleAPI](https://github.com/andresamayadiaz/FrontAccountingSimpleAPI) on GitHub.
